src/PowerShellEditorServices.Host/LanguageServer.cs

```diff
@@ -152,6 +153,46 @@ public void Initialize()
             await requestContext.SendResult(null);
         }
 
+        private async Task HandleExpandAliasRequest(
+            string content,
+            EditorSession editorSession,
+            RequestContext<object, object> requestContext)
+        {
+            var psCommand = new PSCommand();
+            var script = @"
+function Expand-Alias {
+
+    param($targetScript)
+
+    [ref]$errors=$null
+    
+    $tokens = [System.Management.Automation.PsParser]::Tokenize($targetScript, $errors).Where({$_.type -eq 'command'}) | 
+                    Sort Start -Descending
+
+    foreach ($token in  $tokens) {
+        $definition=(Get-Command ('`'+$token.Content) -CommandType Alias -ErrorAction SilentlyContinue).Definition
+
+        if($definition) {        
+            $lhs=$targetScript.Substring(0, $token.Start)
+            $rhs=$targetScript.Substring($token.Start + $token.Length)
+            
+            $targetScript=$lhs + $definition + $rhs
+       }
+    }
+
+    $targetScript
+}
+M
+Expand-Alias @'" + "\r\n" + content + "\r\n'@";
```

Comment:
I wonder if it also makes sense to obfuscate the "Expand-Alias" command somehow so that it doesn't overwrite a similarly-named command that the user has imported/created in their session.  Maybe call it `__Expand-Alias` (two leading underscores)?